plugin/config/src/main/java/com/alibaba/nacos/plugin/config/ConfigChangePluginManager.java

```diff
@@ -83,8 +84,10 @@ private static void loadConfigChangeServices() {
             // map the relationship of pointcut and plugin service
             addPluginServiceByPointCut(each);
```

Comment:
check  service enabled here ，if not enabled，do not add to the final list , and logs should be printed for what we loaded from service loader and what we ignored if service not enabled

Reply:
If you load here, it won't be able to fetch dynamically updated config like AuthConfigs?

Reply:
> If you load here, it won't be able to fetch dynamically updated config like AuthConfigs?
ok, you are right, just ignore this comment